examples/ex14.cpp

```diff
@@ -62,132 +68,154 @@ int main(int argc, char *argv[])
                   "One of the three DG penalty parameters, should be positive."
                   " Negative values are replaced with (order+1)^2.");
    args.AddOption(&eta, "-e", "--eta", "BR2 penalty parameter.");
+   args.AddOption(&pa, "-pa", "--partial-assembly", "-no-pa",
+                  "--no-partial-assembly", "Enable Partial Assembly.");
    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
                   "--no-visualization",
                   "Enable or disable GLVis visualization.");
+   args.AddOption(&device_config, "-d", "--device",
+                  "Device configuration string, see Device::Configure().");
    args.Parse();
    if (!args.Good())
    {
       args.PrintUsage(cout);
       return 1;
    }
+   args.PrintOptions(cout);
    if (kappa < 0)
    {
       kappa = (order+1)*(order+1);
+      cout << "Using kappa = " << kappa << endl;
    }
-   args.PrintOptions(cout);
```

Comment:
For consistency, it's good to make this change in `ex14p` as well. Or, if you prefer, you can revert this change.

Reply:
Reverted in d42dd9c0a